### DISCUSSION TITLE
Rolllade Stop nur aktiv bei Activity

### DISCUSSION THREAD

**Discussion by Sineos** [Zefau/ioBroker.jarvis#2054] [Q&A]
Ich verwende einen Tasmota Rollladenaktor, bekomme es aber nicht hin, dass Stop nur aktiv ist, wenn die Rolllade auch in Bewegung ist. Konfiguriert ist das so:

![image](https://user-images.githubusercontent.com/4459312/202023615-e13abeea-3e4f-4a43-b1ba-cc431d5c0474.png)

Direction ist 0 bei keiner Aktivität und 1 bzw. -1 bei hoch und runter
Was mache ich falsch?

**Comment by Zefau** [comment]
Die Booleans dürfen nicht in `'`, sonst sind es Strings. Und hast du BlindLevelAction auf `value` umgestellt?

**Comment by Sineos** [comment]
Die Hochkommas waren nur ein Test, ob sich was ändert. Leider hat das Umstellen auf `value` auch nichts gebracht. Die komplette Definition ist wie folgt:

 ```
 "eg_blind_wz_8361d_7f5c7": {
    "name": "blind_ug_jamie",
    "icon": "",
    "label": "Rolllade Jamie",
    "function": "blind",
    "states": {
      "level": {
        "state": "sonoff.0.TM_JamieBlind.Shutter1_Position",
        "action": "sonoff.0.TM_JamieBlind.ShutterPosition1",
        "actionElement": "BlindLevelAction",
        "stateKey": "level",
        "showState": false,
        "BlindLevelActionConfig": {
          "val": "value"
        }
      },
      "reachable": {
        "stateKey": "reachable",
        "showState": true,
        "state": "sonoff.0.TM_JamieBlind.alive"
      },
      "activity": {
        "stateKey": "activity",
        "showState": true,
        "state": "sonoff.0.TM_JamieBlind.Shutter1_Direction",
        "properties": {
          "value": "val => val==0 ? false : true"
        },
        "actionElement": null,
        "IconStateConfig": {
          "val": "value"
        }
      },
      "stop": {
        "action": "",
        "actionElement": "IconButtonAction",
        "stateKey": "stop",
        "state": "sonoff.0.TM_JamieBlind.ShutterStop"
      }
    },
    "options": {},
    "attributes": {
      "imported": true,
      "manufacturer": {
        "name": "HomeMatic / HomeMatic IP via CCU",
        "namespace": "hm-rpc"
      }
    },
    "id": "eg_blind_wz_8361d_7f5c7",
    "revision": "8336c"
  },
```

**Comment by mcuiobroker** [comment]
Wie ist der Stand?

**Reply by Sineos** [reply]
Unverändert. Der Aktor funktioniert tadellos. Das Einzige ist, dass Stop immer aktiv ist.

**Reply by mcuiobroker** [reply]
Wie ist die Einstellung im BlindLevelAction? val oder value?
Im Layout und Gerät gleich?
Activity und Level gesetzt im BlindLevelAction? Lt. Einstellung oben nicht?

Warum unterschiedliche DPs?
![image](https://user-images.githubusercontent.com/84818934/202930165-1aa6b56f-150b-4e73-87ec-6d053669e4c2.png)
